src/components/BorrowerProfile/JumpLinks.vue

```diff
@@ -3,6 +3,7 @@
 		<hr class="tw-hidden md:tw-block tw-border-tertiary">
 		<a
 			class="tw-hidden md:tw-inline tw-text-primary tw-pr-1 tw-font-medium tw-cursor-pointer"
+			data-testid="bp-jump-link-borrower-story"
```

Comment:
When addressing these you'll want to use their parent context as well. ie. `bp-lend-cta-jump-links` and `bp-jump-link-borrower-story` OR `bp-summary-card-jump-links` and `bp-jump-link-borrower-story`